package.json

```diff
@@ -44,5 +44,11 @@
     "stylelint-scss": "3.19.0",
     "typescript": "4.1.2"
   },
-  "simple-pre-commit": "npx lint-staged"
+  "simple-pre-commit": "npx lint-staged",
+  "dependencies": {
+    "@hookform/resolvers": "^1.3.5",
+    "@types/yup": "^0.29.11",
+    "react-hook-form": "^6.15.4",
+    "yup": "^0.32.9"
+  }
```

Comment:
We it in the root package.json?
Remove it (*only by `npm install <package-name>` command*)